website/package-lock.json

```diff
@@ -1,8 +1,931 @@
 {
   "name": "website",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
nit: What cause the changes to this file?

Reply:
unsure? something in the makefile?

Reply:
Alright, nbd